yarn.lock

```diff
@@ -0,0 +1,1063 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
+
+
+accepts@~1.3.4:
```

Comment:
Not sure if this was regenerated?  I still see references to `express` and other dependencies that I don't think should be here since we're only depending on **eslint** now.